### PR TITLE
Debug/fix crashing issue

### PR DESCRIPTION
**Pull request by cmanamela**
Updated alerts, Segmentleak and faultysensor components.
Please review

### CODE REVIEW

File: waterlog/src/Components/AlertTable.js [hunk 1]
```diff
@@ -1,66 +1,39 @@
...
-        month = '' + (d.getMonth() + 1),
-        day = '' + d.getDate(),
-        year = d.getFullYear();
-
-    if (month.length < 2) month = '0' + month;
-    if (day.length < 2) day = '0' + day;
-
-    return [month, day, year].join('/');
-}
+import { formatDate, getStatusIcon } from './../utils'
```
Comment (ash-rabbit) [inline waterlog/src/Components/AlertTable.js]: add a semi-colon at the end of this import

**Review by ash-rabbit — CHANGES_REQUESTED** [review]

File: waterlog/src/actions/SegmentLeaksByIdActions.js [hunk 2]
```diff
@@ -1,29 +1,23 @@
-import {
-  FETCH_SEGMENTS_LEAKS_BEGIN,
-  FETCH_SEGMENTS_LEAKS_SUCCESS,
-  FETCH_SEGMENTS_LEAKS_FAILURE
-} from './Types';
-import { Globals } from './../Globals';
+import { FETCH_SEGMENTS_LEAKS_BEGIN, FETCH_SEGMENTS_LEAKS_SUCCESS, FETCH_SEGMENTS_LEAKS_FAILURE } from './Types';
```
Comment (FredJEhlers) [inline waterlog/src/actions/SegmentLeaksByIdActions.js]: This was correct over multiple lines. 
please change it back

File: waterlog/src/actions/TankLevelsByIdAction.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import {
     FETCH_TANK_LEVEL_FAILURE,
     FETCH_TANK_LEVEL_SUCCESS
 } from './Types';
-import { Globals } from './../Globals';
+
```
Comment (FredJEhlers) [inline waterlog/src/actions/TankLevelsByIdAction.js:6]: No need for the extra whitespace

File: waterlog/src/reducers/AlertsReducer.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import {
+    FETCH_ALERTS_BEGIN,
+    FETCH_ALERTS_SUCCESS,
+    FETCH_ALERTS_FAILURE
```
Comment (FredJEhlers) [inline waterlog/src/reducers/AlertsReducer.js:4]: inconsistent with indentation used here `waterlog/src/Components/SegmentLeak.js`
choose which indentation you are going to use and stick with it throughout the project

File: waterlog/src/utils.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+		case 'high':
+			return HighStatusIcon();
+		case 'low':
+			return LowStatusIcon();
+		case 'medium':
+			return MediumStatusIcon();
+		default:
+			return null;
+	}
+};
```
Comment (FredJEhlers) [inline waterlog/src/utils.js]: empty line at EOF

Comment (cmanamela) [reply waterlog/src/utils.js]: Done

File: waterlog/src/reducers/AlertsReducer.js (hunk 4 above)
Comment (cmanamela) [reply waterlog/src/reducers/AlertsReducer.js:4]: everything seems to be Ok on this, i dont see the inconsistency

File: waterlog/src/actions/TankLevelsByIdAction.js (hunk 3 above)
Comment (cmanamela) [reply waterlog/src/actions/TankLevelsByIdAction.js:6]: Fixed

File: waterlog/src/actions/SegmentLeaksByIdActions.js (hunk 2 above)
Comment (cmanamela) [reply waterlog/src/actions/SegmentLeaksByIdActions.js]: Fixed

File: waterlog/src/Components/AlertTable.js (hunk 1 above)
Comment (cmanamela) [reply waterlog/src/Components/AlertTable.js]: Fixed

**Review by FredJEhlers — APPROVED** [review]

**Pull request merged**